MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,342 @@
+---
+layout: post
+title: Getting Started with .NET MAUI DataForm control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI DataForm (SfDataForm) control and its basic features.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Getting Started with .NET MAUI DataForm (SfDataForm)
+
+This section provides a quick overview of how to get started with the .NET MAUI DataForm(SfDataForm) for .NET MAUI and also provides a walk-through to configure the .NET MAUI DataForm control in a real-time scenario.
+
+## Creating an application using the .NET MAUI DataForm
+
+1. Create a new .NET MAUI application in Visual Studio.
+
+2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add SfDataForm to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.DataForm] and then install it.
```

Comment:
2. Syncfusion .NET MAUI components are available in the [nuget.org](https://www.nuget.org/). To add the SfDataForm to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.DataForm], then install it.

Reply:
content updated